### ISSUE TITLE
Colored branch stairs ought to be implemented for QoL improvements

### ISSUE THREAD

**Issue by demogorgon22** [Chris-plus-alphanumericgibberish/dNAO#1379]
DNH adds a lot of branches, and it would help distinguish visually to the player which stair is for a branch if the stairs were colored differently than non branch stairs. Several variants add this, and at this point I wouldn't be surprised if 3.7 vanilla does.

**Comment by k21971** [comment]
Agreed. Colored altars would also be nice (vanilla nethack does this now).

**Comment by Chris-plus-alphanumericgibberish** [comment]
I don't want stairs to be differently color when first found, but once the player has been down the staircase branch stairs could be recolored.

**Comment by NeroOneTrueKing** [comment]
This may be useful: https://github.com/NetHack/NetHack/commit/d463a9e2588a97f6d2ae7b49ba559423677d8b76
